source/macos/AdaptiveCards/AdaptiveCards/Base/Targets.swift

```diff
@@ -15,8 +15,12 @@ class ActionOpenURLTarget: NSObject, TargetHandler {
         button.action = #selector(handleButtonAction(_:))
     }
     
+    func configureAction(for actionView: NSView) {
+        delegate?.handleOpenURLAction(actionView: actionView, urlString: url)
+    }
```

Comment:
Can we rename it to `handleSelectionAction` ?
`configureAction` kinda sounds misleading here.
```suggestion
    func handleSelectionAction(for actionView: NSView) {
        delegate?.handleOpenURLAction(actionView: actionView, urlString: url)
    }
```